plugins/src/main/java/opengrok/auth/plugin/ldap/LdapFacade.java

```diff
@@ -197,12 +197,24 @@ private void setWebHooks(WebHooks webHooks) {
      * Go through all servers in the pool and record the first working.
      */
     void prepareServers() {
+        LOGGER.log(Level.FINER, "checking servers for {0}", this);
         for (int i = 0; i < servers.size(); i++) {
             LdapServer server = servers.get(i);
             if (server.isWorking() && actualServer == -1) {
                 actualServer = i;
             }
         }
+
+        // Close the connections to the inactive servers.
+        LOGGER.log(Level.FINER, "closing unused servers");
+        for (int i = 0; i < servers.size(); i++) {
+            if (i != actualServer) {
+                servers.get(i).close();
+            }
+        }
+
+        LOGGER.log(Level.FINER, String.format("server check done (current server: %s)",
```

Comment:
Nit: not wrapping with `if (LOGGER.isLoggable(Level.FINER)) { }` since the argument is a composed string

Reply:
ack